src/test/java/software/amazon/cloudformation/data/create.request.with-extraneous-model-object.json

```diff
@@ -0,0 +1,43 @@
+{
+  "awsAccountId": "123456789012",
+  "bearerToken": "123456",
+  "region": "us-east-1",
+  "action": "CREATE",
+  "responseEndpoint": "https://cloudformation.us-west-2.amazonaws.com",
+  "resourceType": "AWS::Test::TestModel",
+  "resourceTypeVersion": "1.0",
+  "requestContext": {},
+  "requestData": {
+    "callerCredentials": {
+      "accessKeyId": "IASAYK835GAIFHAHEI23",
+      "secretAccessKey": "66iOGPN5LnpZorcLr8Kh25u8AbjHVllv5/poh2O0",
+      "sessionToken": "lameHS2vQOknSHWhdFYTxm2eJc1JMn9YBNI4nV4mXue945KPL6DHfW8EsUQT5zwssYEC1NvYP9yD6Y5s5lKR3chflOHPFsIe6eqg"
+    },
+    "providerCredentials": {
+      "accessKeyId": "HDI0745692Y45IUTYR78",
+      "secretAccessKey": "4976TUYVI234/5GW87ERYG823RF87GY9EIUH452I3",
+      "sessionToken": "842HYOFIQAEUDF78R8T7IU43HSADYGIFHBJSDHFA87SDF9PYvN1CEYASDUYFT5TQ97YASIHUDFAIUEYRISDKJHFAYSUDTFSDFADS"
+    },
+    "providerLogGroupName": "providerLoggingGroupName",
+    "logicalResourceId": "myBucket",
+    "resourceProperties": {
+      "property1": "abc",
+      "property2": 123,
+      "fieldCausesValidationError": {
```

Comment:
So the deserialization will fail if nested properties? But how the validation can successful? There is `additionalProperties: false` for the schema, so additional property should fail on validation.